src/primo/browzine-primo-adapter.js

```diff
@@ -944,6 +1008,11 @@ browzine.primo = (function() {
           libKeyLinkOptimizer.innerHTML += template;
         }
 
+        if (!directToPDFUrl && !articleLinkUrl && !articleRetractionUrl && !articleEocNoticeUrl && problematicJournalArticleNoticeUrl && isArticle(scope) && showExpressionOfConcern()) {
```

Comment:
why do we call "showExpressionOfConcern"? The way i'm reading the code it seems like we're saying "hey we have a problematic journal" so wouldn't we want the "showProblematicJournalUI" function?

Reply:
Good catch @Evroble ! Let me change that and add a test.

I also checked [in Jira](https://thirdiron.atlassian.net/browse/BZ-7970?focusedCommentId=86641) and confirmed that we don't need to have a toggle for turning on or off problematic journals in Discovery Service Adapters.  If that `problematicJournalArticleNoticeUrl` appears in the API, we show it (if there's no retraction or EOC), no other things to check.